10up/distributor
# Github PHPCS workflow  runs only on changed files

### Description of the Change

I updated the GitHub workflow to run only on changed files.


Closes #1074

### How to test the Change

- `composer runs lint` should only run on the `includes` and `templates` directories.
- Github PHPCS workflow should only run on changed PHP files

### Changelog Entry

> Dev - Github PHPCS workflow  runs only on changed PHP files


### Credits

Props @ravinderk 


### Checklist:
